### ISSUE TITLE
Integration with Whatsapp

### ISSUE THREAD

**Issue by nomio19** [erxes/erxes#3772]

**Comment by spatialy** [comment]
Hi, are you working actively on this feature?

Have any ETA?

**Comment by codihaus** [comment]
Yes, we will release it with Whatsapp Business API.
Have no plan for Personal accounts as it violate the policies from Meta

**Comment by spatialy** [comment]
What WAPI method are you releasing, using a 3rd party like Twilio o the new hosted by Meta direct integration option?

**Comment by codihaus** [comment]
Direct access.

Because for self-hosted, it's mainly used for small teams, so, by creating the application with dev mode, only allow specific page is good enough.

Only when we apply to cloud version, then it's needed to have the cloud api access, for SaaS (authenticate on behalf of customer).

Also, by using direct access, for small teams, you do not need to pay extra to 3rd services as well. (Like Twilio or so)

Is my question good enough for you?

**Comment by spatialy** [comment]
Yes, need to wait to test the integration in some use cases we have here to be able to give you all the feedback we can from the experience

**Comment by spatialy** [comment]
Hi @codihaus  @nomio19 have any update on this? Best

**Comment by SirLouen** [comment]
> Is my question good enough for you?

Is this already working in 0.3.39?

**Comment by codihaus** [comment]
Hello, we're still working on this, we will update when it's ready.

**Comment by feliciien** [comment]
can i work on this ?
